forte/data/ontology/top.py

```diff
@@ -1221,11 +1224,22 @@ def payload_index(self) -> int:
 
     @property
     def uri(self) -> Optional[str]:
+        """
+        Universal resource identifier of the data source.
+
+        Returns:
+            Optional[str]: Universal resource identifier of the data source.
+        """
         return self._uri
 
+    def load(self):
+        fn = self.payloading.route(self.meta)
```

Comment:
How to set `self.payloading`? Do we expect users to call `payload.payloading = CustomizedPayloading()` themselves?